plugins/tangle/consistency.go

```diff
@@ -166,6 +177,10 @@ func computeConeDiff(visited map[trinary.Hash]struct{}, tailTxHash trinary.Hash,
 				ledgerChanges := cachedBndl.GetBundle().GetLedgerChanges()
 				for addr, change := range ledgerChanges {
 					coneDiff[addr] += change
+					if math.Abs(coneDiff[addr]) > int64(compressed.TotalSupply) {
+						// referenced bundle is not valid because ledger changes would overflow total supply
+						return nil, ErrRefBundleNotValid
```

Comment:
Make a separate error because it isn't true that the bundle is invalid, it is the cone diff which isn't consistent.

Reply:
We still need to add the bundle in the `refsAnInvalidBundleStorage` right?
So it would look like that?
```diff
        // compute the diff of the cone which the transaction references
	coneDiff, err := computeConeDiff(visited, tailTxHash, tangle.GetSolidMilestoneIndex(), forceRelease)
	if err != nil {
-		if err == ErrRefBundleNotValid {
+		if err == ErrRefBundleNotValid || err == ErrConeDiffNotConsistent {
			// memorize for a certain time that this transaction references an invalid bundle
			// to short circuit validation during a subsequent tip-sel on it again
			PutInvalidBundleReference(tailTxHash)
		}
		return false
	}
```

Reply:
Yeah something like this 👍